integ/test_init/integ_test_stack.py

```diff
@@ -0,0 +1,70 @@
+from aws_cdk import (
+    Stack,
+    aws_autoscaling as autoscaling,
+    aws_ec2 as ec2,
+    aws_ecs as ecs,
+    aws_iam as iam,
+)
+from constructs import Construct
+
+class IntegTestStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+        
+        # change your arn here
+        arn1="arn:aws:s3:::firelens-example-resources/dummy-input.conf"
+        arn2="arn:aws:s3:::firelens-example-resources/dummy-filter.conf"
+        arn3="arn:aws:s3:::firelens-example-resources/dummy-parser.conf"
+        arn4="arn:aws:s3:::firelens-example-resources/dummy-s3-output.conf"
```

Comment:
these could be top level constants instead? I think that makes more sense if the user is supposed to edit this part

Reply:
Yea that makes sense, will modify